### PR TITLE
#5650 the build OS used by custom dockerfile should be configurable

### LINKED ISSUE #5650: the build OS used by custom dockerfile should be configurable
custom dockerfile can configure the "runtime" OS but the build time OS is not configurable :(

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist

- [ ] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [ ] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [ ] Add an assignee (of course, you are the assignee by default)
- [ ] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [ ] Link to an issue fix #5650
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: docker/custom_worker.dockerfile [hunk 1]
```diff
@@ -39,11 +40,9 @@ RUN cp /version $(find "${KAFKA_DIR}" -maxdepth 1 -type d -name "kafka_*")/bin/w
 ARG BRANCH="master"
 ARG COMMIT=$BRANCH
 ARG REPO="https://github.com/oharastream/ohara.git"
-ARG BEFORE_BUILD=""
```
Comment (chia7712) [inline docker/custom_worker.dockerfile:42]: this arg is useless so I remove it

File: docker/custom_worker.dockerfile [hunk 2]
```diff
@@ -55,7 +54,7 @@ RUN cp $(find "/opt/ohara/" -maxdepth 1 -type d -name "ohara-*")/bin/ohara_versi
 # copy connector jars
 RUN cp $(find "/opt/ohara/" -maxdepth 1 -type d -name "ohara-*")/lib/* $(find "${KAFKA_DIR}" -maxdepth 1 -type d -name "kafka_*")/libs/
 
-FROM $OS
+FROM $RUNTIME_OS
 
 # we use wget to download custom plugin from configurator
 RUN apt update && apt install -y wget
```
Comment (jackyoh) [inline docker/custom_worker.dockerfile:60]: 如果 RUNTIME_OS 設定 centos 的作業系統，應該在 build docker image 時會出錯誤。

Comment (chia7712) [reply docker/custom_worker.dockerfile:60]: yep, 假設是使用ubuntu

這邊用ubuntu的原因是ubuntu走得比較快（比起centos），這樣方便我們測試一些新的東西

Comment (jackyoh) [reply docker/custom_worker.dockerfile:60]: OK, I got it.

**Pull request merged**